### PR TITLE
vn/upgrade babel

### PR DESCRIPTION
**Pull request by vickingan**
Upgrade babel to 7+

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@ lib: es6-lib node_modules
 	node_modules/.bin/babel es6-lib --out-dir lib
 
 test: lib
-	GEO_IMPORT_ENV=test ZOOKEEPER_ENSEMBLE="{}" ./node_modules/.bin/mocha es6-test/unit es6-test/smoke --require @babel/register && jshint es6-lib
+	GEO_IMPORT_ENV=test ./node_modules/.bin/mocha es6-test/unit es6-test/smoke --require @babel/register && jshint es6-lib
```
Comment (sjzavaleta) [inline Makefile:6]: Why  does ZOOKEEPER_ENSEMBLE no longer have to be specified?

File: es6-lib/config/index.js [hunk 2]
```diff
@@ -9,23 +9,22 @@
...
+import prod from './prod.js';
+import test from './test.js';
 
 var env = process.env.GEO_IMPORT_ENV;
 if (!env) throw new Error('No GEO_IMPORT_ENV environment set! Please specify one.');
 
 let conf;
 if (env === 'prod') {
-	conf = merge(baseConfig, prodEnv);
+	conf = merge(baseConfig, prod());
```
Comment (sjzavaleta) [inline es6-lib/config/index.js:23]: What's the advantage of these being functions?

**Review by sjzavaleta — APPROVED** [review]
Code seems fine, only have questions

File: Makefile (hunk 1 above)
Comment (vickingan) [reply Makefile:6]: It doesn't have to be specified because we no longer throw errors when `prod.sh` is imported in `es6-lib/config/index.js`.

**Pull request merged**